### PR TITLE
 Rework CosmoScout GUI

### PR DESCRIPTION
**Pull request by octfx**
See https://github.com/cosmoscout/cosmoscout-vr/pull/95

### CODE REVIEW

File: src/LodBody.cpp [hunk 1]
```diff
@@ -57,8 +57,8 @@ LodBody::LodBody(std::shared_ptr<cs::core::GraphicsEngine> const& graphicsEngine
     for (auto const& s : mDEMtileSources) {
       if (s->getName() == val) {
         mPlanet.setDEMSource(s.get());
-        mGuiManager->getSideBar()->callJavascript(
-            "set_elevation_data_copyright", s->getCopyright());
+        mGuiManager->getGui()->callJavascript(
+            "CosmoScout.call", "sidebar", "setElevationDataCopyright", s->getCopyright());
```
Comment (Schneegans) [inline src/LodBody.cpp]: Also in this plugin I would prefer the `callJavascript("CosmoScout.lodBodies.setElevationDataCopyright",...)` syntax.

File: gui/js/lod_body_settings.js [hunk 2]
```diff
@@ -11,12 +11,3 @@ $('#set_enable_auto_terrain_lod').change(function () {
     else
         $("#set_terrain_lod").removeClass("unresponsive");
 });
-
-
-function set_map_data_copyright(text) {
```
Comment (Schneegans) [inline gui/js/lod_body_settings.js:16]: These methods have to be in a `lodBodiesApi`.

**Review by Schneegans — CHANGES_REQUESTED** [review]

File: gui/js/lod_body_settings.js (hunk 2 above)
Comment (octfx) [reply gui/js/lod_body_settings.js:16]: Done in https://github.com/cosmoscout/csp-lod-bodies/pull/8/commits/a843e3046720f24bc848389587ee4b1dec24eea7

File: src/LodBody.cpp (hunk 1 above)
Comment (octfx) [reply src/LodBody.cpp]: Done in https://github.com/cosmoscout/csp-lod-bodies/pull/8/commits/b20752e8ea5980af209ca0ab8f0c430180627203

**Review by Schneegans — APPROVED** [review]

**Comment by Schneegans** [comment]
This can be merged as soon as all related PRs are approved!

**Pull request merged**